tests/test_utils/test_logging.py

```diff
@@ -9,7 +9,7 @@
 import unittest
 from shutil import rmtree
 
-from tmms.utils.logging import tmmsLogger
+from tmms.utils import logging
```

Comment:
"logging" is the name of the builtin Python logging module.  it needs to be renamed in tmms.utils so we don't collide with Python.

Reply:
Fair. But for the different merge request.
Need to submit an ISSUE to not to forget...